cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -81,18 +81,34 @@ func NewCmd() *cobra.Command {
 					return fmt.Errorf("error listing tests %w", err)
 				}
 			} else {
-				scorecardOutput, err = o.RunTests()
+				ctx, cancel := context.WithTimeout(context.Background(), o.WaitTime)
+				defer cancel()
+
+				bundleData, err := o.GetBundleData()
+				if err != nil {
+					return fmt.Errorf("error getting bundle data %w", err)
+				}
+
+				err = o.CreateConfigMap(ctx, bundleData)
+				if err != nil {
+					return fmt.Errorf("error creating ConfigMap %w", err)
+				}
+
+				runner := scorecard.PodTestRunner{}
+				runner.TestConfig = o
+				o.TestRunner = runner
+
+				scorecardOutput, err = o.RunTests(ctx)
 				if err != nil {
 					return fmt.Errorf("error running tests %w", err)
 				}
+
 			}
 
 			return printOutput(outputFormat, scorecardOutput)
 		},
 	}
 
-	scorecardCmd.Flags().StringVarP(&config, "config", "c", "",
-		"path to a new to be defined DSL yaml formatted file that configures what tests get executed")
```

Comment:
I am assuming that because is alpha we can remove the flag whiteout deprecations.

Reply:
correct.